src/main/kotlin/dev/minn/jda/ktx/InlineShardManagerBuilder.kt

```diff
@@ -0,0 +1,542 @@
+/*
+ * Copyright (c) 2021 Florian Spieß
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+@file:Suppress("MemberVisibilityCanBePrivate", "unused", "FunctionName")
+
+package dev.minn.jda.ktx
+
+import com.neovisionaries.ws.client.WebSocketFactory
+import net.dv8tion.jda.api.OnlineStatus
+import net.dv8tion.jda.api.audio.factory.IAudioSendFactory
+import net.dv8tion.jda.api.entities.Activity
+import net.dv8tion.jda.api.hooks.IEventManager
+import net.dv8tion.jda.api.hooks.VoiceDispatchInterceptor
+import net.dv8tion.jda.api.requests.GatewayIntent
+import net.dv8tion.jda.api.sharding.DefaultShardManagerBuilder
+import net.dv8tion.jda.api.sharding.ShardManager
+import net.dv8tion.jda.api.sharding.ThreadPoolProvider
+import net.dv8tion.jda.api.utils.ChunkingFilter
+import net.dv8tion.jda.api.utils.Compression
+import net.dv8tion.jda.api.utils.MemberCachePolicy
+import net.dv8tion.jda.api.utils.SessionController
+import net.dv8tion.jda.api.utils.cache.CacheFlag
+import okhttp3.OkHttpClient
+import java.util.concurrent.ConcurrentMap
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.ScheduledExecutorService
+import java.util.concurrent.ThreadFactory
+import java.util.function.IntFunction
+
+
+/**
+ * Creates a new [ShardManager] instance with the default settings by using [DefaultShardManagerBuilder.createDefault]
+ *
+ * @param token The token to log in with. If set to null, assign a value to [InlineShardManagerBuilder.token].
+ * @param login Whether or not to automatically log in. See: [ShardManager.login]
+ * @param builder The function used to configure the builder.
+ * @return The new [ShardManager] instance.
+ * @see DefaultShardManagerBuilder.createDefault
+ */
+inline fun DefaultShardManager(
+    token: String? = null,
+    login: Boolean = true,
+    builder: InlineShardManagerBuilder.() -> Unit
+                              ) = InlineShardManagerBuilder(DefaultShardManagerBuilder.createDefault(token))
```

Comment:
```suggestion
) = InlineShardManagerBuilder(DefaultShardManagerBuilder.createDefault(token))
```

Reply:
And all the other instances of this. Please adhere to my codestyle.

Reply:
There was no included `.editorconfig`, so I use assumed my own code style as it wasn't /too/ different from yours. But, will do.